.changeset/dirty-coins-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/filters': minor
+---
+
+Implement FilterMenu header component.
```

Comment:
I don't think we need a changeset here, since we aren't publishing `filters` for public consumption until the entire feature is done

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2951/commits/270993db7c82ab5975460c5515973f05899a1e3c
Thank you